docs/src/main/sphinx/installation/cli.rst

```diff
@@ -41,6 +41,27 @@ password value to avoid the prompt.
 Use ``--help`` to see information about specifying the keystore, truststore, and
 other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
 
+External authentication - SSO
+-----------------------------
+
+The  ``--external-authentication`` option is used for browser-based SSO
+authentication, as detailed in :doc:`/security/oauth2`. With this configuration,
+the CLI displays a URL that you have to open in a web browser for authentication.
+
+The expected behaviour is as follows:
+
+ - Start the CLI with the ``--external-authentication`` option
+ - CLI starts and connects to Trino
+ - When first query is submitted, a message to open a browser with a specified URL is displayed by the CLI
+ - Open the URL in a browser and follow through the authentication process
+ - The CLI automatically receives a token from the authentication process
+ - Successful login in the browser, allows the CLI to proceed with executing the query
```

Comment:
Very awkward sentence. I suggest: "When successfully authenticated in the browser, the CLI will proceed to execute the query."

Reply:
This could actually have a couple of different meanings as is, and I think it is the comma that is confusing things. If my above suggestion is incorrect, that is part of the confusion caused by the comma.